.github/workflows/packer-build-push.yaml

```diff
@@ -0,0 +1,48 @@
+name: Packer build
+
+on:
+  push:
+    branches:
+      - "dev"
+
+jobs:
+  deploy:
+    name: Build firefly AMI
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-region: us-east-2
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+    # validate templates
+    - name: Validate Template
+      uses: hashicorp/packer-github-actions@master
+      with:
+        command: validate
+        arguments: -syntax-only
+        target: firefly.pkr.hcl
+
+    # Initialize and install plugins
+    - name: Validate Template
+      uses: hashicorp/packer-github-actions@master
+      with:
+        command: init
+        arguments: -upgrade
+        target: firefly.pkr.hcl
+
+    # build artifact
+    - name: Build Artifact
```

Comment:
Its been a while since i've built an AMI custom. Will this automatically make the AMI available in the build account region?

Reply:
Yes. The AMI is currently available in us-east-2